sounds.json

```diff
@@ -1353,29 +1356,38 @@
     "bedrock_mapping": "PLACE",
     "identifier": "minecraft:copper_grate[waterlogged=false]"
   },
-  "block.copper_grate.hit": {},
-  "block.copper_grate.fall": {},
-  "block.copper_trapdoor.close": {},
-  "block.copper_trapdoor.open": {},
+  "block.copper_grate.hit": {
+    "playsound_mapping": "step.copper_grate",
+    "bedrock_mapping": "HIT"
+  },
+  "block.copper_grate.fall": {
+    "playsound_mapping": "step.copper_grate"
+  },
+  "block.copper_trapdoor.close": {
+    "playsound_mapping": "open_trapdoor.copper"
+  },
+  "block.copper_trapdoor.open": {
+    "playsound_mapping": "open_trapdoor.copper"
+  },
   "block.coral_block.break": {
-    "playsound_mapping": "random.break",
+    "playsound_mapping": "dig.coral",
     "bedrock_mapping": "BREAK_BLOCK"
   },
   "block.coral_block.fall": {
-    "playsound_mapping": "fall.grass",
+    "playsound_mapping": "fall.coral",
     "bedrock_mapping": "FALL"
   },
   "block.coral_block.hit": {
-    "playsound_mapping": "hit.stone",
+    "playsound_mapping": "use.coral",
     "bedrock_mapping": "HIT"
   },
   "block.coral_block.place": {
-    "playsound_mapping": "use.stone",
+    "playsound_mapping": "dig.coral",
```

Comment:
What's the reasoning for not using use.coral here? Just curious; haven't tested it myself

Reply:
On Java, the dig sound is used.
```json
  "block.coral_block.place": {
    "sounds": [
      "dig/coral1",
      "dig/coral2",
      "dig/coral3",
      "dig/coral4"
    ],
    "subtitle": "subtitles.block.generic.place"
  },
  ```